inspirehep/utils/record_getter.py

```diff
@@ -105,3 +106,19 @@ def get_db_record(pid_type, recid):
     from inspirehep.modules.records.api import InspireRecord
     pid = PersistentIdentifier.get(pid_type, recid)
     return InspireRecord.get_record(pid.object_uuid)
+
+
+def get_db_records(pid_type, recids):
```

Comment:
The name is too generic, it gets a db record by pid.

Reply:
I don't disagree, but it's modelled on `get_db_record` and `get_es_records`, both of which were already there and have similar signatures.